json_serializable-3.5.2/README.md

```diff
@@ -26,7 +26,7 @@ Given a library `example.dart` with an `Person` class annotated with
 `@JsonSerializable()`:
 
 ```dart
-import 'package:json_annotation/json_annotation.dart';
+import 'package:json_annotation_3_1_1/json_annotation.dart';
```

Comment:
There are actually tests that read these code snippets in the README, so they had to be updated